util/topgen.py

```diff
@@ -1174,9 +1175,12 @@ def main():
             if topcfg.setdefault("rnd_cnst_seed", new_seed) == new_seed:
                 log.warning(
                     "No rnd_cnst_seed specified, setting to {}.".format(new_seed))
-        strong_random.unsecure_generate_from_seed(
-            ENTROPY_BUFFER_SIZE_BYTES,
-            topcfg["rnd_cnst_seed"])
+        if (os.getenv('USE_BUFFER')):
+            strong_random.load(SRCTREE_TOP / './util/topgen_entropy_buffer.txt')
```

Comment:
same applies above.

Reply:
this would require updating the chip_sim_cfg.hjson file to pass the hardcoded path in using the flag (and the entropy file would also need to be checked in). so we may want it generated on the fly by [both flows](https://github.com/lowRISC/opentitan/issues/15144#issuecomment-1568715772) (either dvsim.py flow or bazel flow)